public/doc/openapi-3-v1.2.json

```diff
@@ -4060,6 +4109,11 @@
             "description": "The ID of the portfolio item associated to the 'before' step",
             "readOnly": true
           },
+          "post_id": {
```

Comment:
To keep the same pattern, should we change to `post_portfolio_item_id` here?

Reply:
Yes, good catch, it should probably be "after_portfolio_item_id", actually.